src/geo/gmaps/gmaps-cartodb-layer-group-view.js

```diff
@@ -84,9 +84,9 @@ var GMapsCartoDBLayerGroupView = function (layerModel, gmapsMap) {
   // lovely wax connector overwrites options so set them again
   // TODO: remove wax.connector here
   _.extend(this.options, opts);
-  GMapsLayerView.call(this, layerModel, gmapsMap);
+  GMapsLayerView.apply(this, arguments);
```

Comment:
I don't like this, I'd rather to be explicit.

Reply:
We were using `arguments` in most of the places related to this, so I thought it was the best idea to keep it consistent through all the files